warframe/worldstate/models/sortie.py

```diff
@@ -0,0 +1 @@
+### I dunno yet
```

Comment:
look, this really sucks, let's do some real code

```suggestion
import * from ./
```